### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by ChangeNext** [AILab-CVC/SEED#14]
Thank you for amazing project. 
Can you provide the training code?

**Comment by geyuying** [comment]
We have released the training code of SEED-LLaMa, including SEED tokenizer, Multimodal LLM pretraining and instruction tuning. Our Multimodal LLM training codebase supports 1. large-scale multi-node training with deepspeed 2. highly-efficient multiple training datapipes.
